cfgov/cfgov/settings/base.py

```diff
@@ -468,6 +468,9 @@
     DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
     MEDIA_URL = os.path.join(os.environ.get("AWS_S3_URL"), AWS_LOCATION, "")
 
+# Prefix for our elasticsearch indexes based on the environment
+ES_INDEX_PREFIX = os.environ.get('ES_INDEX_PREFIX', '')
```

Comment:
Could we use the existing `settings.DEPLOY_ENVIRONMENT` for this instead of making a new setting/environment variable?